src/utils/errors/account.ts

```diff
@@ -37,6 +37,14 @@ const messages = defineMessages({
     id: "TDhHMi",
     defaultMessage: "This needs to be unique",
   },
+  invalidToken: {
+    id: "INfaIc",
+    defaultMessage: "Invalid or expired token",
```

Comment:
Wouldn't user have a problem with understanding the error?
If the error only occurs when user is not logged in, maybe something like the following would be more descriptive?
```suggestion
    defaultMessage: "Invalid or expired token. Please log in to the dashboard first.",
```

Reply:
User lands on this page when someone request reseting password for them. It requires token & email in URL params. What about adding copy below instead ?

```
Invalid or expired token. Please check your token in URL.
```

Reply:
Yes, it looks fine 👍